tests/react-intl_mock.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+jest.mock('react-intl', function() {
+    const reactIntl = jest.requireActual('react-intl');
+    const enMessages = require('i18n/es.json');
```

Comment:
Looks mostly good, but you're loading the Spanish translation strings by mistake
```suggestion
    const enMessages = require('i18n/en.json');
```

Reply:
Ops done